pom.xml

```diff
@@ -1258,7 +1291,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-plugin-plugin</artifactId>
-          <version>3.5</version>
```

Comment:
The plugin version does not always comply with the Maven version. It is too dangerous and might lead to future build failures

Reply:
@bbortt I think this is why the build_jdk11 job is failing